packages/grafana-e2e-selectors/src/selectors/components.ts

```diff
@@ -18,6 +18,7 @@ export const Components = {
     Panel: {
       title: (title: string) => `Panel header title item ${title}`,
       headerItems: (item: string) => `Panel header item ${item}`,
+      containerByTitle: (title: string) => `Panel container title ${title}`,
```

Comment:
Out of curiosity, what's the difference between `title` and `containerByTitle `? I think might need to remove one of them.

Reply:
Container is targeting the panel container so you can access its children in an easy way. The Title is targeting the actual title/header.